build/build.sh

```diff
@@ -55,6 +55,7 @@ if [ "$PUBLISH" == true ]; then
 	TAGS+=(
 		"${IMAGE_TAGS[@]}"
 	)
+	BUILD_ARGS+=("--push")
```

Comment:
was this broken before because this was missing?

Reply:
maybe we can mention this fix in the PR title as this ends up in the `main` branch log

Reply:
Yes, exactly. Tried a test release of an image and things worked as expected, except:

- It only built but didn't push the images, which this fixes
- There was an access error when the script tried to create the GH Release entry which is probably a simple fix, but figured we'd look at that when we get to testing this together

I'll update the title.